WordPress/src/main/java/org/wordpress/android/ui/FilteredRecyclerView.java

```diff
@@ -526,6 +608,89 @@ public boolean isFirstItemVisible() {
         return (child != null && mRecyclerView.getLayoutManager().getPosition(child) == 0);
     }
 
+    class FilteredPagerAdapter extends PagerAdapter {
```

Comment:
I think this could be moved to a separate (kotlin) file. The only reason why this is an inner class is `mSelectingRememberedFilterOnCreate = true;` and you can actually set that from the place that calls `updateFiltersIfNeeded`. I'd try to not add any more inner non-static classes.

Reply:
Yes makes sense. Changed here 3127de41d671c1a72d7047b80c6feac1a387253e 👍